app-crypt/tpm2-openssl/tpm2-openssl-1.1.1.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="OpenSSL Provider for TPM2 integration"
+
+HOMEPAGE="https://github.com/tpm2-software/tpm2-openssl"
+SRC_URI="https://github.com/tpm2-software/tpm2-openssl/releases/download/${PV}/${P}.tar.gz"
+LICENSE="BSD"
+
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+IUSE="test"
+
+RDEPEND=">=app-crypt/tpm2-tss-3.2.0:=
+	=dev-libs/openssl-3.0*:0="
+
+DEPEND="${RDEPEND}
+	test? ( app-crypt/swtpm
+		app-crypt/tpm2-abrmd
+		app-crypt/tpm2-tools )"
+
+BDEPEND="sys-devel/autoconf-archive
```

Comment:
you're not calling (e)autoreconf so autoconf-archive probably isn't needed

Reply:
Ok, I dropped it